stories/button/button/index.js

```diff
@@ -31,14 +31,14 @@ const ButtonStory = withDocs(ButtonReadme, () => (
       <Button buttonType="quaternary">Quaternary Button</Button>
       <Button disabled>Disabled Button</Button>
 
-      <Button loading>Primary Loading</Button>
-      <Button loading buttonType="secondary">
+      <Button isLoading>Primary Loading</Button>
+      <Button isLoading buttonType="secondary">
```

Comment:
I tested that the basic `loading` functionality worked by changing these locally to be `loading` and seeing no regressions.